src/main/java/core/basesyntax/service/operations/BalanceOperationsHandler.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.model.Operations;
+
+public class BalanceOperationsHandler implements OperationsHandler {
```

Comment:
There is an unnecessary empty line at the beginning of the class definition. Please remove it to follow the proper coding conventions.